internal/pkg/instrumentation/bpf/google.golang.org/grpc/server/probe.go

```diff
@@ -184,6 +233,11 @@ func processFn(e *event) ptrace.SpanSlice {
 		}
 	}
 
+	if serverAddr {
+		attrs = append(attrs, semconv.ServerAddress(net.IP(e.LocalAddr.IP[:]).String()))
+		attrs = append(attrs, semconv.ServerPort(int(e.LocalAddr.Port)))
```

Comment:
This `int64`->`int` has the potential for a truncation error. Given ports should always fit in an `int32` can we use an `int32` for the `Port` field instead?

Reply:
Seems to work in my latest commit